trailblazer/store/crud/update.py

```diff
@@ -93,6 +93,32 @@ def _update_analysis_from_slurm_squeue_output(
         analysis.logged_at = datetime.now()
         self.commit()
 
+    def update_tower_run_status(self, analysis_id: int) -> None:
+        """Query tower for entries related to given analysis, and update the Trailblazer database."""
+        analysis: Optional[Analysis] = self.get_analysis_with_id(analysis_id=analysis_id)
+        tower_api: Optional[TowerAPI] = get_tower_api(
+            config_file_path=analysis.config_path, case_id=analysis.family
+        )
+
+        try:
+            self._update_analysis_from_tower_output(analysis, analysis_id, tower_api)
+        except Exception as error:
+            LOG.error(f"Error logging case - {analysis.family} :  {type(error).__name__}")
```

Comment:
Is this enough information for the error log? Maybe it would be better to include the whole error output and not only the name, but feel free to disregard

Reply:
I agree, but do not want to change function in this PR.